tests/MongoDB.Driver.Legacy.Tests/MongoServerTests.cs

```diff
@@ -178,6 +179,7 @@ public void TestGetDatabase()
             var database = __server.GetDatabase("test", settings);
             Assert.Equal("test", database.Name);
             Assert.Equal(ReadPreference.Primary, database.Settings.ReadPreference);
+            Assert.Equal(database._operationExecutor(), __server._operationExecutor());
```

Comment:
Test that the MongoDatabase inherits the _operationExecutor from the MongoServer.